util/py_lib/seqan/dox/write_html.py

```diff
@@ -22,10 +22,11 @@ def escapeForXml(s):
 
 def escapeName(name):
```

Comment:
I can not just use urllib yet, as more complicated things will happen, due to automatic encoding/decoding of URI addresses. The presented hot fix solves the problems for the majority of fraud links due to numbers in the URI name.
The remaining issues (very rare cases), lead to valid addresses due to a bug in the common.js (#831).